00-bootcamp/06-our-model-of-collaboration/helloWorld.js

```diff
@@ -1 +1 @@
-// Your code here.
+// Your code here.
```

Comment:
What `hellWorld.js` and 'codeStyle.js' are doing in pull request ?
please, remove them